extension/packages/nextjs/components/OnchainKitScaffoldProvider.tsx

```diff
@@ -0,0 +1,15 @@
+import { PropsWithChildren } from "react";
+import { OnchainKitProvider } from "@coinbase/onchainkit";
+import { useAccount } from "wagmi";
+
+export const OnchainKitScaffoldProvider = ({ children }: PropsWithChildren) => {
+  const { chain } = useAccount();
+
+  return chain ? (
```

Comment:
Maybe instead of returning empty children, what if we just default the chain to base? 

For example : 

```

  return (
    <OnchainKitProvider chain={chain ? chain : base} apiKey={process.env.NEXT_PUBLIC_ONCHAINKIT_API_KEY}>
      {children}
    </OnchainKitProvider>
  );
};
```

Reply:
Thanks! Updated [fd82574](https://github.com/scaffold-eth/create-eth-extensions/pull/11/commits/fd82574b82887ee757129487f71b1418e5dc03ca)